### ISSUE TITLE
APK project settings not writen

### ISSUE THREAD

**Issue by Alisis33** [radimitrov/CSharpShellApp#161]
The changes for the APK project settings are not writen in the csproj.

**Comment by radimitrov** [comment]
Yes, the are written elsewhere as it doesn't make sense for a Console project to have APK settings. Or for them to be stored in 2 different ways.

**Comment by Alisis33** [comment]
You mean:
package name
Version
ABIs, arm x86
?

All APKs hafe the path
com.companyname.xxx
No matter what I set up.

**Comment by radimitrov** [comment]
Yes. Also that is the format android wants for a package name. Anything else and it won't install, so I automatically adapt for example "projectname" to "com.companyname.projectname". But if you enter for example  "com.alisis.projectname" it will stay that way

**Comment by Alisis33** [comment]
Your test pattern is wrong.
Change company name.
Open a other project.
Open the 1. Again
